README.md

```diff
@@ -87,6 +87,109 @@ with Maven.
 This project uses the [Google java code style](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html)
 to keep code clean and consistant.
 
+Maven Plugin
+-----------
+
+## schema-registry:download
+
+This plugin is used to download AVRO schemas for the requested subjects.
+
+|        Name        |                                                      Description                                                      |  Type  | Required | Default |
+|:------------------:|:---------------------------------------------------------------------------------------------------------------------:|:------:|:--------:|:-------:|
+| schemaRegistryUrls | Urls for the schema registry instance to connect to                                                                   |  List  |    yes   |         |
+| outputDirectory    | Output directory to write the schemas to.                                                                             | String |    yes   |         |
+| schemaExtension    | The file extension to use for the output file name. This must begin with a '.' character.                             | String |    no    |  .avsc  |
+| subjectPatterns    | The subject patterns to download. This is a list of regular expressions. Patterns must match the entire subject name. | List   |    no    |   ^.+$  |
```

Comment:
My thoughts were not necessarily to have this integrated into the phases of a build. More to be used adhoc during development to pull down the schemas and version them with your current project.